scripts/us_census/acs5yr/subject_tables/common/testdata/spec_s2702.json

```diff
@@ -0,0 +1,483 @@
+{
```

Comment:
This file is much too large for a test -- please add a TODO to create a smaller spec for testing